src/containers/Header/Header.tsx

```diff
@@ -33,7 +33,7 @@ function Header() {
         useSelector((state: any) => state);
 
     const clusterName: string = useSelector(
-        (state: any) => state.cluster.data?.Name || state.clusterInfo.title,
+        (state: any) => state.cluster.data?.Name || state.clusterInfo?.title,
```

Comment:
do we really want to keep the usage of a reducer that isn't in the code base anymore?

Reply:
Yes, now it is needed in internal version.

Reply:
Keeping in the open source version something that is only required in the internal version doesn't seem right to me. This selector should accept some parameter, and internal version should use it to fallback to `clusterInfo.title`. And in open source we should only keep `state.cluster`.

I can approve this PR if we agree to change this selector in the future

Reply:
I totally agree with you. It seems not very difficult to achieve this, so I fixed the issue.